### PR TITLE
Externalize User-facing Strings in go-whisk

### PR DESCRIPTION
**Pull request by dubeejw**
- Externalize all user-facing strings in the go-whisk package
- See https://github.com/openwhisk/openwhisk/issues/1118
- See also related PR #1119
- PG1: 342

### CODE REVIEW

**Comment by dubeejw** [comment]
PG1 315.

**Comment by dubeejw** [comment]
@mduser, any comments?

**Comment by dubeejw** [comment]
@mdeuser, any comments?

File: tools/go-cli/go-whisk/whisk/trigger.go [hunk 1]
```diff
@@ -96,15 +99,17 @@ func (s *TriggerService) Insert(trigger *Trigger, overwrite bool) (*TriggerFromS
     routeUrl, err := url.Parse(route)
     if err != nil {
         Debug(DbgError, "url.Parse(%s) error: %s\n", route, err)
-        errStr := fmt.Sprintf("Invalid request URL '%s': %s", route, err)
+        errStr := fmt.Sprintf(wski18n.T("Invalid request URL '{{.url}}': {{.err}}",
+            map[string]interface{}{"route": route, "err": err}))
```
Comment (mdeuser) [inline tools/go-cli/go-whisk/whisk/trigger.go]: PII label mismatch {{.url}} vs "route".

**Comment by mdeuser** [comment]
@dubeejw - Were you able to run that PII test tool against this code to confirm all the wski18n.T calls work?

**Comment by dubeejw** [comment]
@mdeuser, the test tool did not pick up a lot of the strings. I think it's just the regex isn't correct though.

**Comment by dubeejw** [comment]
@mdeuser, this should be ready for you to merge after Travis passes.

**Pull request merged**